### DISCUSSION TITLE
Custom Templates can't find "_row.twig"

### DISCUSSION THREAD

**Discussion by Larsundso** [solspace/craft-freeform#1616] [Questions & Answers]
In the middle of updating from 4.x to 5.x, I already did migration and am now in the process of transforming my templates.

I've followed the docs for creating custom templates by copying the bootstrap-5 index.twig into my custom template file.
Trying to open render the page now throws this error:

> The template “@freeform-formatting-templates/bootstrap-5/_row.twig” could not be found in “__string_template__3bf363610c5199ee9fa1577b3b758587” at line 9.

and

> The template “freeform/_templates/formatting/bootstrap-5/_row.twig” could not be found in “__string_template__3bf363610c5199ee9fa1577b3b758587” at line 9.

I tried changing the import on the first error, that didn't work.
How can i fix this?

**Comment by kjmartens** [comment]
Duplicate of https://github.com/solspace/craft-freeform/discussions/1658
